lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -663,16 +664,25 @@ class ProgressPageTests(ModuleStoreTestCase):
 
     def setUp(self):
         super(ProgressPageTests, self).setUp()
+        SelfPacedConfiguration(enabled=True).save()
```

Comment:
You should have a test that disables `SelfPacedConfiguration` and checks query counts as well, just to make sure that that doesn't change the query counts.